src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -187,7 +187,7 @@ class EthService(
 
   lazy val blockchain = BlockchainImpl(blockchainStorages)
 
-  val minerTimeOut: Long = 5.seconds.toMillis
+  val minerTimeOut: Long = 5.seconds.toMillis // TODO: configure timeout
```

Comment:
I'm not aware of it, but it was discussed the other day. It's not the scope of this PR, though, so I imagine another task will be planned ( @mcsherrylabs )